src/modules/didRegistry/didRegistry.validation.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  DIDAttribute,
+  Encoding,
+  PubKeyType,
+} from '@ew-did-registry/did-resolver-interface';
+import { KeyType } from '@ew-did-registry/keys';
+import { createCheckers, iface, enumtype } from 'ts-interface-checker';
+
+const IS_REQ_STRING = 'string';
```

Comment:
What is the purpose of this alias?

Reply:
as name says if we got 'property': 'string' in interface-checker , it indicates that this field is of type 'string' and it's required. const is not to repeat constant strings across document and avoid typos